packages/@okta/vuepress-site/docs/guides/sign-in-with-google/configure-idp/index.md

```diff
@@ -6,13 +6,13 @@ title: Configure Google as an Identity Provider in Okta
 
 2.2. On the main page, click on the **Admin** button in the upper right.
 
-2.3. Hover your cursor over **Users** until the menu opens, then click on **Social & Identity Providers**.
+2.3. If you use the Developer Console, hover your cursor over **Users** until the menu opens, then click on **Social & Identity Providers**. If you use the Classic UI, hover over **Security** then click on **Identity Providers**.
 
 2.4. On the Identity Providers page, click on **Add Identity Provider** > **Add Google**.
 
 * **Name:** We suggest using the name you would expect to see on a button, something like "Log in to Google".
-* **Client Id:** Paste in the App ID that you got from Google in step 1.3 above.
-* **Client Secret:** Paste in the App Secret that you got from Google in step 1.3 above.
+* **Client Id:** Paste in the OAuth client ID that you got from Google in step 1.3.
+* **Client Secret:** Paste in the OAuth client secret that you got from Google in step 1.3.
```

Comment:
**Client Secret:** Paste the OAuth client secret that you got from Google in step 1.3.